lib/UnsafeCachePlugin.js

```diff
@@ -15,7 +15,11 @@ UnsafeCachePlugin.prototype.apply = function(resolver) {
 	var regExps = this.regExps;
 	var cache = this.cache;
 	resolver.resolve = function resolve(context, request, callback) {
-		var id = context + "->" + request;
+		var requestHash = request;
+		if(typeof requestHash === 'object' && requestHash.hasOwnProperty('path')) {
+			requestHash = request.path;
```

Comment:
@sokra That's true, thanks for the hint. I've updated it to use `JSON.stringify`.